app/javascript/components/widgets/widgets/forest-change/glad-ranked/selectors.js

```diff
@@ -66,8 +66,7 @@ export const parseData = createSelector(
       return {
         id: k,
         color: colors.main,
-        percentage:
-          countsAreaPerc >= 0.1 ? `${format('.2r')(countsAreaPerc)}%` : '<0.1%',
```

Comment:
Do we definitely want to remove this?

Reply:
yes, it is unused. Although i've found another bug elsewhere so keep this open.